### PR TITLE
1.1 Documentation changes

### PR DESCRIPTION
**Pull request by DukeDavis12**
1.1 Documentation changes.

Signed-off-by: Davis Benny <davis.benny@intel.com>

### CODE REVIEW

File: docs/implementation-references/getting-started-guide.md [hunk 1]
```diff
@@ -192,8 +199,8 @@ Once the Owner instance has successfully started, the following output is displa
 
 !!!NOTES
       - Proper [keystore management](#keystore-management) to be considered before using the services in production environment.
-      - To0scheduling interval property can be modified in the component-sample/demo/owner/owner.env.
-      Update `owner_to0_scheduling_interval=30`
+      - To0scheduling interval property can be modified in the component-sample/demo/owner/service.yml.
+      Update to0-scheduler: `interval: 120`
```
Comment (v-vasavi) [inline docs/implementation-references/getting-started-guide.md:203]: Do we need to talk about AIO as  a package and how they can use individual components also?

File: docs/implementation-references/getting-started-guide.md [hunk 2]
```diff
@@ -206,6 +213,9 @@ Once the Owner instance has successfully started, the following output is displa
 
    On a new console, key in the following commands
 
+!!!NOTES
+    Make sure that `di-url` is set to the correct address in `service.yml` of device. The default `di-url` value for standalone manufacturer is `http://localhost:8039`.
```
Comment (v-vasavi) [inline docs/implementation-references/getting-started-guide.md:217]: Can we set this when we release ?

File: docs/implementation-references/getting-started-guide.md [hunk 3]
```diff
@@ -217,7 +227,7 @@ Expect the following line on successful DI completion.
 DI complete, GUID is <guid>
 ```
 
-**After completion of DI, the FDO credentials are stored into `credential.bin` file. The credentials file includes `rvinfo` from manufacturer, which is later used by device to contact RV server, once it is powered on at the client side. The initialized device is then boxed and sold to customers.**
+**After completion of DI, the FDO credentials are stored into `credentials.bin` file. The credentials file includes `rvinfo` from manufacturer, which is later used by device to contact RV server, once it is powered on at the client side. The initialized device is then boxed and sold to customers.**
```
Comment (v-vasavi) [inline docs/implementation-references/getting-started-guide.md:230]: Should we call out database references?

File: docs/implementation-references/getting-started-guide.md [hunk 4]
```diff
@@ -237,17 +247,33 @@ DI complete, GUID is <guid>
...
+Following the steps to manually initiate the T00.
+
+```
+curl -D - --digest -u ${api_user}:${owner_api_passwd} --location --request GET "http://${owner_ip}:${onr_port}/api/v1/certificate?alias=${attestation_type}" -H 'Content-Type: text/plain' -o owner_cert.txt
+
+curl -D - --digest -u ${api_user}:${mfg_api_passwd} --location --request POST "http://${mfg_ip}:${mfg_port}/api/v1/mfg/vouchers/${serial_no}" --header 'Content-Type: text/plain' --data-raw  "<content-of-owner_cert>" -o ${serial_no}_voucher.txt)
+
+curl -D - --digest -u ${api_user}:${onr_api_passwd} --location --request POST "http://${onr_ip}:${onr_port}/api/v1/owner/vouchers/" --header 'Content-Type: text/plain' --data-raw "$extended_voucher" -o ${serial_no}_guid.txt)
+
+curl -D - --digest -u ${api_user}:${onr_api_passwd} --location --request GET "http://${onr_ip}:${onr_port}/api/v1/to0/${device_guid}" --header 'Content-Type: text/plain')
```
Comment (v-vasavi) [inline docs/implementation-references/getting-started-guide.md:268]: Should we call out APIs?

File: docs/implementation-references/getting-started-guide.md [hunk 5]
```diff
@@ -283,7 +308,7 @@ Wait for TO2 protocol completed message and Device is onboarded Successfully.
 Expect the following message on successful TO2 completion.
 
 ```
-TO2 complete, GUID is d43c6dc6...
+TO2 completed successfully.
```
Comment (v-vasavi) [inline docs/implementation-references/getting-started-guide.md:311]: Aren't we adding the GUID here? I thought that was one of the changes requested as part of automation

File: docs/implementation-references/proxy-settings.md [hunk 6]
```diff
@@ -2,6 +2,126 @@
 
 This document lists the different proxy settings that need to be set. 
 
+## Script to automate proxy setup.
```
Comment (v-vasavi) [inline docs/implementation-references/proxy-settings.md:5]: Is this our internal script?

Comment (DukeDavis12) [reply docs/implementation-references/proxy-settings.md:5]: Yes. Removed all internal proxy details.

File: docs/implementation-references/getting-started-guide.md (hunk 2 above)
Comment (DukeDavis12) [reply docs/implementation-references/getting-started-guide.md:217]: Yes. We can.

**Review by v-vasavi — APPROVED** [review]

**Review by rftemple — APPROVED** [review]

File: docs/implementation-references/getting-started-guide.md (hunk 5 above)
Comment (DukeDavis12) [reply docs/implementation-references/getting-started-guide.md:311]: Not part of the latest TO2 completion logging.

**Pull request merged**